clients/python/integration-test.sh

```diff
@@ -18,7 +18,7 @@ trap cleanup EXIT
 docker compose -f docker-compose.test.yml up --build -d
 
 export CHROMA_INTEGRATION_TEST_ONLY=1
-export CHROMA_API_IMPL=chromadb.api.fastapi.FastAPI
+export CHROMA_API_IMPL="${CHROMA_API_IMPL:-chromadb.api.fastapi.FastAPI}"
```

Comment:
learning: why do we need this?

Reply:
(unnecessary when async/sync thin client tests are moved to the same script)

sets the variable to a default if it isn't specified